src/java/us/kbase/mobu/runner/ModuleRunner.java

```diff
@@ -211,13 +240,19 @@ public void logNextLine(String line, boolean isError) {
                             //do nothing, SDK callback server doesn't use a logger
                         }
                     }).build();
-            ModuleRunVersion runver = new ModuleRunVersion(
-                    new URL("https://fakefakefakefakefake.com"),
-                    new ModuleMethod("use_set_provenance.to_set_provenance_for_tests"),
-                    "aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa", "0.0.0", "dev");
+            Set<String> releaseTags = new TreeSet<String>(mv.getReleaseTags());
+            System.out.println("Release tags: " + releaseTags);
+            String requestedRelease = null;
+            final ModuleRunVersion runver = new ModuleRunVersion(
```

Comment:
Why is requested release null? It's supplied on the command line (well, that or a git commit).